### DISCUSSION TITLE
Frankenphp development container keeps crashing for our developers

### DISCUSSION THREAD

**Discussion by SimonMacIntyre** [php/frankenphp#899] [General]
@dunglas Hi Kevin, apologies, I was reluctant to make a new issue as I did see a number of mostly resolved/closed issues mention this. 

This would occasionally happen for 20% of our developers now and then. 

Suddenly this morning with fresh images, its even worse, it is happening for 90% of developers and almost immediately after startup. 

We are using Docker image: `dunglas/frankenphp:1-php8.3-alpine`, and this is how we start the server:
```
CMD [ "/bin/sh", "-c", "php artisan octane:start --server=frankenphp ${CADDYFILE:+--caddyfile=$CADDYFILE} --host=$HOST --port=$PORT --workers=$WORKERS --max-requests=$MAX_REQUEST ${HTTPS:+--https} ${HTTP_REDIRECT:+--http-redirect} ${WATCH:+--watch} --admin-port=$ADMIN_PORT"]

```

I also swapped this to `octane:frankenphp` but the same results. 

Oddly enough, I myself am having a hard time reproducing, but it is consistently breaking immediately for everyone else. We're all on macs, so the images are `arm64`. I cannot figure out why it is not reproduced by me, but everyone else is consistently breaking. It was mostly fine though even just yesterday.

Should I make a new issue? Any ideas?

![image](https://github.com/dunglas/frankenphp/assets/1260938/fea03704-3664-4db7-8302-5e242122b119)

**Comment by SimonMacIntyre** [comment]
This is our latest image sha: dunglas/frankenphp:1-php8.3-alpine@sha256:835f439c3843e40871ae39a73871b44a2d7af48ab8e56700109076750d23f4a9

**Comment by dunglas** [comment]
This is very likely this problem: https://github.com/golang/go/issues/68285
A workaround is to use the Debian image for now.
